src/SFA.DAS.PensionsRegulator.Database/StoredProcedures/GetOrganisationsByPAYEReferenceAndAORN.sql

```diff
@@ -0,0 +1,20 @@
+﻿CREATE PROCEDURE [dbo].GetOrganisationsByPAYEReferenceAndAORN
+	@PAYESchemeReference varchar(25),
+	@AORN varchar(25)
+AS
+SELECT 
+Organisation.OrganisationName as Name,
+Organisation.Status as Status,
+Organisation.TPR_Unique_Id as UniqueIdentity,
+OrganisationAddress.AddressLine1 as AddressLine1,
+OrganisationAddress.AddressLine2 as AddressLine2,
+OrganisationAddress.AddressLine3 as AddressLine3,
+OrganisationAddress.AddressLine4 as AddressLine4,
+OrganisationAddress.AddressLine5 as AddressLine5,
+OrganisationAddress.PostCode as PostCode
+FROM
```

Comment:
As, I presume, this stored proc and GetOrganisationsByPAYEReference are doing the same joins and using the same columns would it be worth creating a view and simplifying the stored procs?

Reply:
View created an sprocs modified to read from view.